packages/common/src/schemas/documents/workflow/documents/schemas/GH.ts

```diff
@@ -38,20 +38,20 @@ export const getGhanaDocuments = (): TDocument[] => {
         timeRun: Type.Optional(Type.String()),
       }),
     },
-    {
-      category: 'financial_information',
-      type: 'mtn_statement_businesses',
-      issuer: { country: 'GH' },
-      issuingVersion: 1,
-      version: 1,
-      propertiesSchema: Type.Object({
-        msisdn: Type.String({ pattern: '^233[0-9]{9}$' }),
-        accountHolderName: TypeStringAtLeastOneWord,
-        from: Type.String({ format: 'date' }),
-        to: Type.String({ format: 'date' }),
-        timeRun: Type.String({ format: 'date-time' }),
-      }),
-    },
+    // {
+    //   category: 'financial_information',
+    //   type: 'mtn_statement_businesses',
+    //   issuer: { country: 'GH' },
+    //   issuingVersion: 1,
+    //   version: 1,
+    //   propertiesSchema: Type.Object({
+    //     msisdn: Type.String({ pattern: '^233[0-9]{9}$' }),
+    //     accountHolderName: TypeStringAtLeastOneWord,
+    //     from: Type.String({ format: 'date' }),
+    //     to: Type.String({ format: 'date' }),
+    //     timeRun: Type.String({ format: 'date-time' }),
+    //   }),
+    // },
```

Comment:
Fido needs MTN state both for Fido Score and Fido Biz, but they are not the same, so we added this new document with `_businesses` suffix. You need verify with Fido that they don't need this document.